app/views/home/home.html.erb

```diff
@@ -1,7 +1,6 @@
 <% cache('feature_home-intro', skip_digest: true) do %>
   <%= feature("home-intro") %>
 <% end %>
-
```

Comment:
Ah i see in the final diff nothing has changed. Thank you, just checking! Merging!